staging/flagger/templates/rbac.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.create }}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
api changes and new label feature, should this be a minor version chart bump at least?